README.md

```diff
@@ -6,24 +6,101 @@
+    + [Bitbucket Server](#bitbucket-server)
+      - [Command-line arguments](#command-line-arguments-2)
+      - [Derived docker image](#derived-docker-image-2)
+    + [GitLab](#gitlab)
+      - [Command-line arguments](#command-line-arguments-3)
+      - [Derived docker image](#derived-docker-image-3)
+    + [Azure Repos](#azure-repos)
+      - [Command-line arguments](#command-line-arguments-4)
+      - [Derived docker image](#derived-docker-image-4)
+    + [Artifactory](#artifactory)
+      - [Command-line arguments](#command-line-arguments-5)
+      - [Derived docker image](#derived-docker-image-5)
+    + [Nexus 3](#nexus-3)
+      - [Command-line arguments](#command-line-arguments-6)
+      - [Derived docker image](#derived-docker-image-6)
+    + [Jira](#jira)
+      - [Command-line arguments](#command-line-arguments-7)
+      - [Derived docker image](#derived-docker-image-7)
+    + [Container registry agent](#container-registry-agent)
+      - [Command-line arguments](#command-line-arguments-8)
+      - [Derived docker image](#derived-docker-image-8)
+    + [Monitoring](#monitoring)
+      - [Health Check](#health-check)
+      - [System Check](#system-check)
+      - [Logging](#logging)
+    + [Advanced Configuration](#advanced-configuration)
+      - [HTTPS](#https)
+      - [Git with an internal certificate](#git-with-an-internal-certificate)
+      - [Infrastructure as Code (IaC)](#infrastructure-as-code--iac-)
+      - [Changing the Auth Method](#changing-the-auth-method)
+    + [Credential Pooling](#credential-pooling)
+      - [Limitations](#limitations)
+      - [Credentials Matrix](#credentials-matrix)
+    + [Custom approved-listing filter](#custom-approved-listing-filter)
+      - [Types of Filtering](#types-of-filtering)
+        * [By Header](#by-header)
+    + [Mounting Secrets](#mounting-secrets)
+    + [Troubleshooting](#troubleshooting)
+      - [Support of big manifest files (> 1Mb) for GitHub / GitHub Enterprise](#support-of-big-manifest-files----1mb--for-github---github-enterprise)
+  * [Misc](#misc)
+
+<small><i><a href='http://ecotrust-canada.github.io/markdown-toc/'>Table of contents generated with markdown-toc</a></i></small>
+
+## Overview
+
+The Broker consists of two parts
+
+- **Broker Client**: A Docker image deployed in your infrastructure.
+- **Broker Server**: An application deployed in Snyks infrastructure.
+
+(as the Broker Server is provided by Snyk and no installation is required on your part, all further descriptions in this README refer only to the Broker Client unless stated otherwise)
+
+The Broker Client establishes a websocket 'tunnel' to the Broker Server, proxying requests from snyk.io to your SCM (fetching manifest files from monitored repositories), and vice versa (webhooks posted by the SCM). Because this tunnel is established by the Broker Client making an outbound request, there is no need to grant permissions to specific Snyk IP addresses in your firewall.
```

Comment:
It's not strictly for SCMs - For example, Artifactory, Nexus, and Jira aren't SCM systems. I'm not sure what the preferred term would be instead though - possibly "backend system"?

Reply:
OK, I'll update and do another PR.

Probably change the top section to reference "SCM or other backend system, such as Jira or Artifactory" and then use "backend system" throughout the rest of the doc.

BTW, any issues with leaving the small "ToC created by {third party}" link in there? It's auto-added by the ToC generator, and I can remove it if Snyk policy is to not reference stuff like that... At the same point, I *did* use that ToC generator, so...